docs/pages/setup/operations/backup-restore.mdx

```diff
@@ -217,75 +215,13 @@ Of particular note:
     
 </TabItem>
 <TabItem scope={["cloud"]} label="Teleport Cloud">
-<Notice type="tip">
 
-In Teleport Cloud, backend data is managed for you automatically. Follow the
-instructions below if you wish to migrate your Teleport Cloud backend to a
-self-hosted one.
+In Teleport Cloud, backend data is managed for you automatically. If you would
+like to migrate configuration resources to a self-hosted Teleport cluster,
+follow our recommended backup practice of storing configuration resources in a
+git repository and creating them regularly. This will enable you to keep your
```

Comment:
I'm not sure if this should say "creating them regularly" or "updating them regularly".

Reply:
Added another suggestion that's less ambiguous